### PR TITLE
added media section and carousel

### PR DESCRIPTION
**Pull request by srikayy**
please pull my request....added carousel and stuff

### CODE REVIEW

File: -staging-/index.html [hunk 1]
```diff
@@ -76,7 +79,7 @@
                         <!--TODO: Get that style tag out of there-->
                     </li>
                     <li>
-                        <a class="page-scroll" href="#" onclick="alert('The SB Robotics PA website is under construction. Check back soon!','SBRPA')">Parents</a>
+                        <a class="page-scroll" href="#" onclick="alert('T6he SB Robotics PA website is under construction. Check back soon!','SBRPA')">Parents</a>
```
Comment (matticoli) [inline -staging-/index.html]: please fix this, will cause merge conflict

File: -staging-/index.html [hunk 2]
```diff
@@ -127,6 +130,78 @@
         </div>
     </section>
     
+    
+    <!-- Media Section -->
+    <section id="media" class="container-fluid content-section text-center">
+        <div class="row">
+            
+                <h2>Media</h2>
+<!--                Littttttt-->
```
Comment (matticoli) [inline -staging-/index.html]: remove this

File: -staging-/index.html [hunk 3]
```diff
@@ -127,6 +130,78 @@
...
+						<div id="myCarousel" class="carousel slide" data-ride="carousel" style="border-radius: 10px">
+							<!-- Indicators -->
+							<ol class="carousel-indicators">
+								<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+								<li data-target="#myCarousel" data-slide-to="1"></li>
+								<li data-target="#myCarousel" data-slide-to="2"></li>
+								<li data-target="#myCarousel" data-slide-to="3"></li>
+								<li data-target="#myCarousel" data-slide-to="4"></li>
+								<li data-target="#myCarousel" data-slide-to="5"></li>
+                                <li data-target="#myCarousel" data-slide-to="6"></li>
```
Comment (matticoli) [inline -staging-/index.html]: fix indentations

File: -staging-/stylesheets/grayscale.css [hunk 4]
```diff
@@ -401,6 +401,12 @@ body {
   -ms-transform: rotate(0deg);
   transform: rotate(45deg);
 }
+.caro {
```
Comment (matticoli) [inline -staging-/stylesheets/grayscale.css]: you can leave this for now, but it's slightly ambiguous

File: -staging-/stylesheets/grayscale.css [hunk 5]
```diff
@@ -385,7 +385,7 @@ body {
   border-right: 2px solid #eee;
   border-bottom: 2px solid #eee;
   border-radius: 2px;
-  color: #eee;
+  color: #eee;
```
Comment (matticoli) [inline -staging-/stylesheets/grayscale.css]: don't need that space there

**Comment by srikayy** [comment]
mikel review

**Review by matticoli — CHANGES_REQUESTED** [review]
lg overall, see comments.

**Review by matticoli — APPROVED** [review]

**Pull request merged**